src/commons/sagas/AchievementSaga.ts

```diff
@@ -192,6 +192,7 @@ export default function* AchievementSaga(): SagaIterator {
       (state: OverallState) => state.session.enableAchievements
     );
     if (action.payload.find(e => e === EventType.ERROR)) {
+      // TODO update this to work with new side content system
```

Comment:
It's not been done because I'm not sure what the intended behaviour is supposed to be, or how best to go about integrating it (like what should actually be responsible for the handling of this specific alert)